airflow/providers/fab/CHANGELOG.rst

```diff
@@ -20,7 +20,33 @@
 Changelog
 ---------
 
-1.0.0
+1.0.2
 .....
 
-Initial version of the provider.
+First stable release for the provider
+
+
+.. Below changes are excluded from the changelog. Move them to
+   appropriate section above if needed. Do not delete the lines(!):
+   * ``Upgrade FAB to 4.4.1 (#38319)``
+   * ``Bump ruff to 0.3.3 (#38240)``
+   * ``Make the method 'BaseAuthManager.is_authorized_custom_view' abstract (#37915)``
+   * ``Avoid use of 'assert' outside of the tests (#37718)``
+   * ``Resolve G004: Logging statement uses f-string (#37873)``
+   * ``Remove useless methods from security manager (#37889)``
+   * ``Use 'next' when redirecting (#37904)``
+   * ``Add "MENU" permission in auth manager (#37881)``
+   * ``Avoid to use too broad 'noqa' (#37862)``
+   * ``Add post endpoint for dataset events (#37570)``
+   * ``Add "queuedEvent" endpoint to get/delete DatasetDagRunQueue (#37176)``
+   * ``Add swagger path to FAB Auth manager and Internal API (#37525)``
+   * ``Revoking audit_log permission from all users except admin (#37501)``
+   * ``Enable the 'Is Active?' flag by default in user view (#37507)``
+   * ``Add comment about versions updated by release manager (#37488)``
+   * ``Until we release 2.9.0, we keep airflow >= 2.9.0.dev0 for FAB provider (#37421)``
+   * ``Improve suffix handling for provider-generated dependencies (#38029)``
```

Comment:
not listing all features/bug fixes on user facing release notes as this is actually the first stable release so everything we do prior to release is not interesting for users.